action/dependencies.py

```diff
@@ -63,13 +64,23 @@ def add_packages(packages: str):
         raw string from github action, syntax defined in README.md
     """
 
+    if len(packages) == 0 or packages.isspace():
```

Comment:
```suggestion
    if packages.strip() == '':
```
Just a suggestion, I don't really care either way.